conmon/conmon.c

```diff
@@ -370,18 +354,21 @@ static int write_k8s_log(int fd, stdpipe_t pipe, const char *buf, ssize_t buflen
 			fd = log_fd;
 		}
 
-		/* Output a newline */
-		if (insert_newline) {
-			if (writev_buffer_append_segment(fd, &bufv, "\n", -1) < 0) {
-				nwarn("failed to write newline to log");
+		/* Output the timestamp */
+			if (writev_buffer_append_segment(fd, &bufv, tsbuf, -1) < 0) {
+				nwarn("failed to write (timestamp, stream) to log");
 				goto next;
 			}
-		}
 
-		/* Output a timestamp */
-		if (insert_timestamp) {
-			if (writev_buffer_append_segment(fd, &bufv, tsbuf, -1) < 0) {
-				nwarn("failed to write (timestamp, stream) to log");
+		/* Output log tag for partial or newline */
+		if (partial) {
+			if (writev_buffer_append_segment(fd, &bufv, "P ", -1) < 0) {
```

Comment:
Is something upstream from conmon interpreting this "E " and "P "?

Reply:
Yeah, the linked kubernetes PR does that. We need to wait for that to be finalized and merged. I just wanted to make sure that we can handle this in conmon.